source/common/tls/default_tls_certificate_selector.cc

```diff
@@ -180,24 +180,20 @@ DefaultTlsCertificateSelector::findTlsContext(
   const Ssl::TlsContext* candidate_ctx = nullptr;
   Ssl::OcspStapleAction ocsp_staple_action;
 
+  int cec_count = client_ecdsa_capabilities.size();
```

Comment:
This isn't needed; just use `client_ecdsa_capabilities.empty()`. Or if it's clearer, make this a bool. But you never need the actual number of entries.